From ad54489f907569acd7a7d2cc47fa2699dea98707 Mon Sep 17 00:00:00 2001
From: yu fang <yu.fang@amlogic.com>
Date: Wed, 2 Jan 2019 16:32:36 +0800
Subject: [PATCH] TvSettings: Do not release mHandler in onStop. [1/1]

PD#SWPL-3645

Problem:
Crashed when return from wifi details to NetworkFragment.

Solution:
Do not release mHandler in onStop.

Verify:
Verified on all platforms.

Change-Id: If6cf038e9ad26dde6ef4aa13a0f0e488529d0619
Signed-off-by: yu fang <yu.fang@amlogic.com>
---
 Settings/src/com/android/tv/settings/connectivity/NetworkFragment.java | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Settings/src/com/android/tv/settings/connectivity/NetworkFragment.java b/Settings/src/com/android/tv/settings/connectivity/NetworkFragment.java
index 4e5d7d3..31f940d 100644
--- a/Settings/src/com/android/tv/settings/connectivity/NetworkFragment.java
+++ b/Settings/src/com/android/tv/settings/connectivity/NetworkFragment.java
@@ -146,8 +146,6 @@ public class NetworkFragment extends SettingsPreferenceFragment implements
     @Override
     public void onStop() {
         super.onStop();
-        mHandler.removeCallbacksAndMessages(null);
-        mHandler = null;
         mContentResolver.unregisterContentObserver(mNetworkObserver);
     }

--
1.9.1

